src/components/AnonymousChat/child/ChatItemText.style.ts

```diff
@@ -165,5 +171,6 @@ export const textStyle = (isMyText: boolean) => [
 export const messageStyle = (isMyText: boolean, messageType?: string) => [
   styles.text,
   isMyText ? {color: colors.white} : {color: colors.black},
-  messageType === 'deleted' && styles.deletedText
+  isMyText && messageType === MESSAGE_TYPE_DELETED && styles.deletedMyText,
+  !isMyText && messageType === MESSAGE_TYPE_DELETED && styles.deletedTargetText
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change introduces separate handling for deleted messages from the user and the target. It's a good approach but ensure that the new constant `MESSAGE_TYPE_DELETED` is defined correctly and used consistently across the codebase.
```diff
-  messageType === 'deleted' && styles.deletedText
+  isMyText && messageType === MESSAGE_TYPE_DELETED && styles.deletedMyText,
+  !isMyText && messageType === MESSAGE_TYPE_DELETED && styles.deletedTargetText
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->